wg-etcd-operator/charter.md

```diff
@@ -0,0 +1,60 @@
+# WG etcd operator
+
+This charter adheres to the conventions described in the [Kubernetes Charter README]
+and uses the Roles and Organization Management outlined in [sig-governance].
+
+[sig-governance]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md
+[Kubernetes Charter README]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+The purpose of an etcd-operator is to operate automatically etcd clusters which run in the Kubernetes environment.
```

Comment:
Maybe something in the lines of:
```suggestion
The purpose of an etcd-operator is to operate automatically etcd clusters which run in the Kubernetes environment (with the exclusion of the etcd backing kubernetes cluster).
```

Reply:
Thanks for the suggestion. Updated, it should be clearer now. PTAL.